pkg/machines/components/vm/disks/vmDisksCard.jsx

```diff
@@ -234,7 +234,7 @@ export class VmDisksCard extends React.Component {
         }
 
         const rows = disks.map(disk => {
-            const idPrefixRow = `${idPrefix}-${disk.target || disk.device}`;
+            const idPrefixRow = `${idPrefix}-${(disk.target || disk.device)}`;
```

Comment:
Second-guessing my JS fu: This is just purely stylistic, no behaviour change, right?

Reply:
Indeed, this does not change anything -  sorry for the messy diff. I will revert to how it was if I will repost.